wiki/markdown/curl-examples.md

```diff
@@ -281,16 +282,30 @@ The response is a json object containing all the connected components and data a
         <td>/aggregated-objects/query</td>
         <td>POST</td>
         <td>Free style query; user specified queries</td>
-    </tr>
+    </tr>    
+</table>
+
+Example curl commands to these endpoints [can be found here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/query-aggregated-objects.md) and [here](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/failed-notifications.md)
```

Comment:
You can remove the link to failed-notifications.md now

Reply:
done